modules/linear_boltzmann_solvers/discrete_ordinates_solver/sweep/fluds/aah_fluds_common_data.cc

```diff
@@ -347,30 +347,30 @@ AAH_FLUDSCommonData::LocalIncidentMapping(const Cell& cell,
   std::vector<std::pair<int, std::vector<short>>> inco_face_dof_mapping;
 
   // Loop over faces but process only incident faces
-  for (short f = 0; f < cell.faces.size(); ++f)
+  for (auto f = 0; f < cell.faces.size(); ++f)
   {
     const CellFace& face = cell.faces[f];
     const auto& orienation = spds.CellFaceOrientations()[cell.local_id][f];
 
-    //%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%% Incident face
+    // Incident face
     if (orienation == FaceOrientation::INCOMING)
     {
       if (face.IsNeighborLocal(grid))
       {
         // Find associated face for dof mapping
-        int ass_face = cell_nodal_mapping[f].associated_face_;
+        auto ass_face = cell_nodal_mapping[f].associated_face_;
```

Comment:
here's another ass face.  `assoc_face`  might be preferred?

Reply:
Fixed.